Description: adapt to crossbuild
 updating build to make it compatible with crossbuilding a package
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2025-11-23
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/JUCE/extras/Build/juceaide/CMakeLists.txt
+++ b/JUCE/extras/Build/juceaide/CMakeLists.txt
@@ -111,6 +111,10 @@
             "-DCMAKE_BUILD_TYPE=Debug"
             "-DJUCE_BUILD_HELPER_TOOLS=ON"
             "-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}"
+            "-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}"
+			"-DCMAKE_SYSROOT=${CMAKE_SYSROOT}"
+			"-DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}"
+			"-DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}"
             ${extra_compiler_flag_arguments}
         WORKING_DIRECTORY "${JUCE_SOURCE_DIR}"
         OUTPUT_VARIABLE command_output
